testsuite/features/support/env.rb

```diff
@@ -221,6 +248,7 @@ def process_code_coverage
 
 # Dump feature code coverage into a Redis DB before we run next feature
 Before do |scenario|
+  check_read_timeout_threshold
```

Comment:
I think we don't need to run it here, as we are already running in another `Before` method. If we add it here too, we will repeating that check multiple times before running a scenario.